composer.json

```diff
@@ -45,8 +45,7 @@
         "symfony/finder": "^5.3.7|^6.0",
         "symfony/mailer": "^5.1|^6.0",
         "symfony/translation": "^5.1|^6.0",
-        "vlucas/phpdotenv": "^5.4",
-        "nyholm/psr7": "^1.5.0"
+        "vlucas/phpdotenv": "^5.4"
```

Comment:
Try to add `"psr/http-factory-implementation": "^1.0"` please

Reply:
Added